Content.Shared/Chat/SharedChatSystem.cs

```diff
@@ -246,6 +246,8 @@ public static string InjectTagInsideTag(ChatMessage message, string outerTag, st
     public static string InjectTagAroundString(ChatMessage message, string targetString, string tag, string? tagParameter)
     {
         var rawmsg = message.WrappedMessage;
+        targetString = targetString.Replace("s", "s*");
+        targetString = targetString.Replace("z", "z*");
```

Comment:
`*` means "zero or more". You definitely want `+` if you're gonna use this approach.

Reply:
You are right. I've updated it with the correct operator